ansible_runner/defaults.py

```diff
@@ -4,3 +4,8 @@
 
 # for ansible-runner worker cleanup command
 GRACE_PERIOD_DEFAULT = 60  # minutes
+
+# values passed to tempfile.mkdtemp to generate a private data dir
+# when user did not provide one
+auto_create_naming = '.ansible-runner-'
+auto_create_dir = None
```

Comment:
Since these are constants, these should be all caps.
```suggestion
AUTO_CREATE_NAMING = '.ansible-runner-'
AUTO_CREATE_DIr = None
```